lib/vagrant-registration/action/register.rb

```diff
@@ -11,7 +11,7 @@ def initialize(app, _)
         end
 
         def call(env)
-          @app.call(env)
+          @app.call(env) unless Plugin.vbguest_plugin?
```

Comment:
We should include some comment about running the middleware on different places.